lib/networking/vpc/index.ts

```diff
@@ -118,8 +118,13 @@ export class Vpc extends Construct {
     // All HTTP VPC traffic -> ECS model ALB
     ecsModelAlbSg.addIngressRule(Peer.ipv4(vpc.vpcCidrBlock), Port.tcp(80), 'Allow VPC traffic on port 80');
 
-    // All HTTPS IPV4 traffic -> REST API ALB
-    restApiAlbSg.addIngressRule(Peer.anyIpv4(), Port.tcp(443), 'Allow any traffic on port 443');
+    if (config.restApiConfig.internetFacing) {
```

Comment:
should this be based on the existence of the cert or not on the rest api alb? i can still have a cert on a private/internal ALB, which i'd probably want to listen on 443.

Reply:
Good call. I'll make that adjustment.